apps/cloud_composer/include/pcl/apps/cloud_composer/tools/impl/organized_segmentation.hpp

```diff
@@ -103,18 +103,12 @@
      std::vector<pcl::PointIndices> boundary_indices;
      mps.segmentAndRefine (regions, model_coefficients, inlier_indices, labels, label_indices, boundary_indices);
      
-     std::vector<bool> plane_labels;
-     plane_labels.resize (label_indices.size (), false);
-     for (size_t i = 0; i < label_indices.size (); i++)
-     {
-       if (label_indices[i].indices.size () > min_plane_size)
-       {
-         plane_labels[i] = true;
-       }
-     }
+     boost::shared_ptr<std::map<uint32_t, bool> > plane_labels = boost::make_shared<std::map<uint32_t, bool> > ();
+     for (size_t i = 0; i < label_indices.size (); ++i)
+       (*plane_labels)[i] = (label_indices[i].indices.size () > min_plane_size)? true : false;
```

Comment:
Just noticed I did something useless here (and the other repetitions of this pattern). This can be reduced to:
```cpp
(*plane_labels)[i] = (label_indices[i].indices.size () > min_plane_size);
```
🤦‍♂️

I'll submit the change at some point.